### ISSUE TITLE
[mac os][bug] Couldn't launch the game with last update 

### ISSUE THREAD

**Issue by nesterow** [redeclipse/base#909]
MacOS Mojave 14.6
I don't know what libsteam license is, but I thought you can link it statically.

Console output:
```
bins: 2e9ede6132ca3ee7748c6919e7be15db6b638924 is the current version.
dyld: Library not loaded: @rpath/libsteam_api.dylib
  Referenced from: /Applications/redeclipse.app/Contents/Resources/bin/redeclipse.app/Contents/MacOS/redeclipse_universal
  Reason: image not found
Abort trap: 6
logout
...bla-bla-bla
```
- Version and build: [2e9ede6132ca3ee7748c6919e7be15db6b638924]

**Comment by qreeves** [comment]
I'm guessing this is `stable` (v1.6) and not `master`? I'll have to look at why this keeps happening on my macOS build server.

**Comment by nesterow** [comment]
It should be on stable, I use official package for MacOS.

**Comment by qreeves** [comment]
Restructured the build setup on my macOS system to resolve these issues. Should be fixed as of e149d9d65873b0200aa72d964cd87dd0d9cc1118 - let me know how it goes.

**Comment by nesterow** [comment]
[e149d9d65873b0200aa72d964cd87dd0d9cc1118] works as expected.
